inc/saiport.h

```diff
@@ -2290,6 +2290,9 @@ typedef enum _sai_port_stat_t
     /** PRBS Error Count */
     SAI_PORT_STAT_PRBS_ERROR_COUNT,
 
+    /** PRBS Receive Error Count */
+    SAI_PORT_STAT_PRBS_RX_ERROR_COUNT,
```

Comment:
what is the existing SAI_PORT_STAT_PRBS_ERROR_COUNT?

Reply:
Removed this attribute
https://github.com/opencomputeproject/SAI/pull/1179/commits/ce4f8429862738e2707537cd3459a30a08912321

Also introduced the complex data type for handling atomic state as discussed in the meeting